### PR TITLE
Added improvements related to languages column

### PR DESCRIPTION
**Pull request by leonofre**
Related to #80

### CODE REVIEW

File: admin/class-monk-admin.php [hunk 1]
```diff
@@ -529,6 +529,10 @@ public function monk_language_column_content( $column_name, $post_id ) {
 			$active_languages     = get_option( 'monk_active_languages', false );
 			$post_type            = get_query_var( 'post_type' );
 			$post_type            = isset( $post_type ) && ! empty( $post_type ) ? sanitize_text_field( wp_unslash( $post_type ) ) : false;
+			$avaliable_languages  = false;
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Variable misspeling: 
`$available_languages` not `$avaliable_languages`

File: admin/class-monk-admin.php [hunk 2]
```diff
@@ -662,6 +672,24 @@ public function monk_taxonomy_language_column_content( $content, $column_name, $
...
+					$avaliable_languages = true;
+				}
+			}
+
+			foreach ( $taxonomies as $taxonomy ) {
+				if ( isset( $_GET['taxonomy'] ) ) {
+					if ( $_GET['taxonomy'] === $taxonomy ) {
+						$base_url     = admin_url( 'term.php?taxonomy=' . $taxonomy );
+						$new_term_url = add_query_arg( array(
+								'monk_term_id' => $monk_term_translations_id,
```
Comment (brenoalvs) [inline admin/class-monk-admin.php:688]: This parameter will be modified to `monk_id`

File: admin/partials/admin-taxonomy-monk-language-column.php [hunk 3]
```diff
@@ -53,7 +43,9 @@
 			<?php endif; ?>
 		<?php endforeach; ?>
 	<?php endforeach; ?>
-	<a class="monk-new-post-link" href="<?php echo esc_url( $new_term_url ); ?>">Add +</a>
+	<?php if ( $avaliable_languages ) : ?>
+		<a class="monk-new-translation-link" href="<?php echo esc_url( $new_term_url ); ?>">Add +</a>
```
Comment (brenoalvs) [inline admin/partials/admin-taxonomy-monk-language-column.php]: Localize label `__( 'Add+', 'monk' )`

File: admin/partials/monk-language-column.php [hunk 4]
```diff
@@ -39,7 +37,9 @@
 			<?php endif; ?>
 		<?php endforeach; ?>
 	<?php endforeach; ?>
-	<a class="monk-new-post-link" href="<?php echo esc_url( $new_post_url ); ?>">Add +</a>
+	<?php if ( $avaliable_languages ) : ?>
+		<a class="monk-new-translation-link" href="<?php echo esc_url( $new_post_url ); ?>">Add +</a>
```
Comment (brenoalvs) [inline admin/partials/monk-language-column.php]: Localize label __( 'Add+', 'monk' )

File: admin/partials/monk-language-column.php [hunk 5]
```diff
@@ -39,7 +37,9 @@
...
+	<?php endif; ?>
 <?php
```
Comment (brenoalvs) [inline admin/partials/monk-language-column.php]: Fix `<?php else : ?>` code standard

**Review by brenoalvs — CHANGES_REQUESTED** [review]

File: admin/class-monk-admin.php (hunk 2 above)
Comment (leonofre) [reply admin/class-monk-admin.php:688]: This is related to issue #79

**Review by brenoalvs — APPROVED** [review]

**Pull request merged**